index.bs

```diff
@@ -475,7 +475,8 @@ When this method is invoked, the user agent MUST execute the following algorithm
     1. If |extension| is not supported by this client platform, then [=continue=].
 
     1. Otherwise, let |result| be the result of running |extension|'s [=client processing=] algorithm on |argument|. If the
-        algorithm returned an error, [=continue=].
+        algorithm returned an error, return [=a promise rejected with=] a {{DOMException}} whose name is
+        "{{InvalidParameterError}}", and terminate this algorithm.
```

Comment:
Good catch that this DOMException name doesn't exist yet. The proper way to add a DOMException is to send a PR against https://heycam.github.io/webidl/#idl-DOMException-error-names with an explanation of why none of the existing names fits this case. I think in this case a `TypeError` might make sense though.

I think this algorithm change makes sense overall. If a known exception has a bad format, it does make sense to report that.